ffigen/linux.yaml

```diff
@@ -10,12 +10,15 @@ headers:
     - 'realm.h'
     - 'realm_dart.h'
     - 'realm_dart_scheduler.h'
+    - 'realm_android_platform.h'
```

Comment:
bit surprised to see `android` here.. and similar for macos and windows, but I assume it is intended (just checking)

Reply:
yes this is needed. this is the linux host config. In other words  if we want to run the ffigen on linux this is the config to use. It should have these headers to generate correct bindings